node/pkg/aggregator/types.go

```diff
@@ -45,19 +45,19 @@ type globalAggregate struct {
 
 type App struct {
 	Bus         *bus.MessageBus
-	Aggregators map[int64]*AggregatorNode
+	Aggregators map[int64]*Aggregator
```

Comment:
do you think adding `Active` prefix makes it more understandable? I'm not sure because this type also include `isRunning` which stands for state telling if it is active or not

Reply:
`active` is used from db table standing for permanent state, and `isRunning` is used within application standing for ephemeral state, do you have any nice suggestions? it's not easy to catch this concept by its name

Reply:
can I just stick to name without prefix then?